components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/PostAuthnMissingClaimHandler.java

```diff
@@ -125,6 +125,7 @@ public PostAuthnHandlerFlowStatus handle(HttpServletRequest request, HttpServlet
                             handlePostAuthenticationForMissingClaimsRequest(request, response, context);
                     return flowStatus;
                 }
+                throw e;
```

Comment:
thrown the top-level exception back, this is a regression caused by this change. https://github.com/wso2/carbon-identity-framework/pull/3382/files#diff-c9c0f305a1b7a215f87b01e97e9ab380703414e4c6bada58df5b85c2b630bac7R120